package.json

```diff
@@ -37,6 +37,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
혹시 이거 왜 설치하셨나용? @kebab000 @RyuChaewon

Reply:
One of your dependencies, babel-preset-react-app, is importing the
"@babel/plugin-proposal-private-property-in-object" package without
declaring it in its dependencies. This is currently working because
"@babel/plugin-proposal-private-property-in-object" is already in your
node_modules folder for unrelated reasons, but it may break at any time.

babel-preset-react-app is part of the create-react-app project, which
is not maintianed anymore. It is thus unlikely that this bug will
ever be fixed. Add "@babel/plugin-proposal-private-property-in-object" to
your devDependencies to work around this error. This will make this message
go away.


이 메시지는 프로젝트의 의존성 중 하나인 babel-preset-react-app이 '@babel/plugin-proposal-private-property-in-object' 패키지를 의존성으로 선언하지 않고 가져오고 있다는 것을 알려줍니다. 현재는 작동하지만 이는 언제든지 오동작할 수 있는 위험이 있습니다.

해결 방법으로는 babel-preset-react-app에 의존성으로 '@babel/plugin-proposal-private-property-in-object'를 명시적으로 추가하는 것을 제안합니다. 이렇게 함으로써 미래에 발생할 수 있는 오류를 방지할 수 있습니다.

라고 합니당